packages/core/CHANGELOG.md

```diff
@@ -7,120 +7,154 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.0.16](https://github.com/waku-org/js-waku/compare/core-v0.0.15...core-v0.0.16) (2023-04-03)
 
-
 ### ⚠ BREAKING CHANGES
 
-* add and implement IReceiver ([#1219](https://github.com/waku-org/js-waku/issues/1219))
+- add and implement IReceiver ([#1219](https://github.com/waku-org/js-waku/issues/1219))
```

Comment:
`Changelogs` should be reverted.
Did it happen after you got my fix for prettier?